guides/hack/06-classes/18-constants.md

```diff
@@ -9,17 +10,24 @@ class Automobile {
 
 <<__EntryPoint>>
 function main(): void {
-  $col = Automobile::DEFAULT_COLOR;
+  $col = Automobile::DEFAULT_COLOR; // or: $col = self::DEFAULT_COLOR;
   echo "\$col = $col\n";
 }
 ```
 
-If a class constant's type is omitted, it is inferred from the initializer, which must be present. In this case, that type is `string`.
+## Visibility
+Class constants have public visibility and can not be declared as `public`, `protected`, or `private`.
+
+## Selection
+Inside a parent class, use `self::foo` to access a named constant `foo`. Outside a parent class, a class constant's name must be fully qualified with the class and constant name (e.g. `Bar::foo`). For more information, see [scope-resolution operator, `::`](../expressions-and-operators/scope-resolution.md).
```

Comment:
Can you normalize this link to match the path?

If you prefer the `../` form, we probably need to improve the internal link test to be able to verify these

Reply:
Sure, didn't even notice the different format. Will update.